inclusive-design/wecount.inclusivedesign.ca
# chore(deps): run npm update

## Description

I figured we should run `npm update` and make sure we have incorporated any security fixes to our dependencies (there's some audit warnings on our production branch).

## Steps to test

1. Check the deploy preview.

**Expected behavior:** Everything works as expected.

## Additional information

Not applicable.

## Related issues

Not applicable.
